src/api/app/views/layouts/webui/_login_form.html.haml

```diff
@@ -1,22 +1,9 @@
-- if can_signup
+- if can_sign_up?
   %li.nav-item
-    = link_to('Sign Up', signup_url, class: 'nav-link')
+    = link_to('Sign Up', sign_up_params[:url], class: 'nav-link')
 %li.nav-item.dropdown#login-form-dropdown
   = link_to('#', id: 'login-trigger', class: 'nav-link dropdown-toggle', 'data-toggle': 'dropdown') do
     Log In
   .dropdown-menu.dropdown-menu-right.shadow-lg.bg-dark.top-of-flash{ 'aria-labelledby': 'dropdownMenuButton' }
     .px-4.py-3#login-form
-      = form_tag(form_url, options) do
-        - if proxy
-          = hidden_field_tag(:context, 'default')
-          = hidden_field_tag(:proxypath, 'reserve')
-          = hidden_field_tag(:message, 'Please log in')
-          = hidden_field_tag(:url, request.fullpath)
-        .form-group
-          = label_tag(:username, 'Username', class: 'text-light')
-          = text_field_tag(:username, nil, placeholder: 'User Name', required: true, class: 'form-control')
-        .form-group
-          = label_tag(:password, 'Password', class: 'text-light')
-          = password_field_tag(:password, nil, placeholder: 'Password', required: true, class: 'form-control')
-        .clearfix
-          = submit_tag('Log In', class: 'btn btn-success float-right')
+      = render partial: 'webui/session/form', locals: { text_color: 'text-light', submit_position: 'float-right', with_sign_up: false }
```

Comment:
I would suggest to rename `text_color` to `label_css`, and `submit_position` to `button_css`.

Reply:
Thanks!, done